php-cs-fixer.ta.php

```diff
@@ -1,13 +1,22 @@
 <?php
 
+use PhpCsFixer\RuleSet\Sets\PhpCsFixerSet;
+
+$csFixerSet = new PhpCsFixerSet();
+
 return (new PhpCsFixer\Config('standards'))
     ->setRiskyAllowed(true)
     ->setRules([
         '@PhpCsFixer' => true,
         '@Symfony:risky' => true,
```

Comment:
I think at some point we have to drop the `@PhpCsFixer` ruleset: it has a very large overlap with the `@Symfony` ruleset, but has some different configuration for some rules. This is kind of confusing, especially when we look up the rules in https://mlocati.github.io/php-cs-fixer-configurator/